distributed/worker.py

```diff
@@ -2449,7 +2452,16 @@ def _maybe_deserialize_task(self, ts):
             return ts.runspec
         try:
             start = time()
-            function, args, kwargs = _deserialize(*ts.runspec)
+            # Offload deserializing large tasks
+            offload_threshold = get_offload_threshold()
+            if sizeof(ts.runspec) > offload_threshold:
```

Comment:
I want to think about if there's a better way to do this. This may lead to a slowdown if we start submitting task deserialization to a busy offloading thread pool.